### DISCUSSION TITLE
'npm install' gets stuck and eventually 'Killed' at something called 'swc...'

### DISCUSSION THREAD

**Discussion by azakero** [vercel/next.js#33121] [Help]
Okay so I have been trying to deploy a Next app to my Ubuntu 20.04 vps but I am unable to do so. I have uploaded all the files from my local machine to the server via ftp and then after trying to install it gets stuck and then killed ( please check the image to see where it gets stuck ). The app works perfectly in my local machine.

I tried by deleting .next directory, node_modules directory as well as package-lock.json file but the problem prevails.

Here's the Node and NPM version in my local machine:

Node: v14.15.0
NPM: v7.14.0

This is the Node and NPM version in my server:

Node: v14.18.2
NPM: v8.3.0

I have also attached my package.json file incase you want to see the dependencies

![Screenshot_3](https://user-images.githubusercontent.com/49979539/148700140-aae9478b-5dfb-4fcc-b10e-01b69d979378.jpg)
![Screenshot_4](https://user-images.githubusercontent.com/49979539/148700141-dd759fb7-06de-45cd-a177-5e1b77d3e60a.jpg)

**Comment by Neilxx** [comment]
I solved with the solution here: [npm install via Digital Ocean gets killed](https://stackoverflow.com/questions/49228066/npm-install-via-digital-ocean-gets-killed)
